databases/week2/dbweek2.sql

```diff
@@ -0,0 +1,74 @@
+/* week 2 homework */ 
+
+/* working with tasks */ 
+
+USE hyf;
+SELECT * FROM task t ;
+INSERT INTO task (title, description, created, updated, due_date, status_id, user_id)
+VALUES ('study', 'do homework', '2022-02-11', '2022-02-11', '2022-02-15', 2, 4); 
+
+UPDATE task 
+SET title = 'its not all about money'
+WHERE id = 2;
+
+UPDATE task 
+SET due_date = '2017-12-25'
+WHERE id = 2;
+
+/* not sure what is the dif btwn Change a task status
```

Comment:
They're almost the same thing, except if you change the status to Done, you set the status_id to a specific value, or if you want to be super cool, you do a sub-query to find the status_id for status.name='Done' :)